src/js/components/Select/SelectContainer.js

```diff
@@ -4,33 +4,37 @@ import styled, { withTheme } from 'styled-components';
 import {
   debounce,
   debounceDelay,
-  isNodeAfterScroll,
-  isNodeBeforeScroll,
   selectedStyle,
   setFocusWithoutScroll,
 } from '../../utils';
 
 import { defaultProps } from '../../default-props';
 
 import { Box } from '../Box';
+import { Button } from '../Button';
 import { InfiniteScroll } from '../InfiniteScroll';
 import { Keyboard } from '../Keyboard';
 import { Text } from '../Text';
 import { TextInput } from '../TextInput';
 
-import { SelectOption } from './SelectOption';
 import { StyledContainer } from './StyledSelect';
 
 // position relative is so scroll can be managed correctly
-const OptionsBox = styled(Box)`
+const OptionsBox = styled.div`
   position: relative;
   scroll-behavior: smooth;
+  overflow: auto;
 `;
 
 const OptionBox = styled(Box)`
   ${props => props.selected && selectedStyle}
 `;
 
+const SelectOption = styled(Button)`
+  display: block;
+  width: 100%;
+`;
```

Comment:
Since this is replacing the SelectOption original component, I'm wondering if it meets the same accessibility requirements, previous SelectOption Button had:
```        
        tabIndex="-1"
        role="menuitem"
```

Reply:
Those are done via DOM element attributes in the places where SelectOption is instantiated below.